liberapay/utils/unconfusable.py

```diff
@@ -0,0 +1,11 @@
+from confusable_homoglyphs import confusables
+
+def unconfusable_string(name):
```

Comment:
It would be nice to have a docstring explaining what this function does.

Reply:
Done here: a73af92

Tell me if there is a specific format for that.